client/X11/xf_graphics.c

```diff
@@ -562,6 +562,11 @@ static BOOL xf_Pointer_SetPosition(rdpContext* context, UINT32 x, UINT32 y)
 	WLog_DBG(TAG, "%s: %" PRIu32 "x%" PRIu32, __func__, x, y);
 	if (xfc->remote_app && !xfc->focused)
 		return TRUE;
+	else if (!xfc->remote_app)
+	{
+		if (!xfc->focused)
```

Comment:
Like in #9099 combine this condition with the one above to just `if (!xfc->focused)`

Reply:
updated from upstream pr